errata_tool/erratum.py

```diff
@@ -616,65 +616,95 @@ def addCC(self, email):
     # errata-tool, but users can add their own as well.
     #
     def addFlags(self, flags):
-        if type(flags) is not list:
+        if not isinstance(flags, list):
             flags = [flags]
         # Two loops intentionally. First one is for
         # input validation.
         for f in flags:
-            if type(f) is not str:
+            if not isinstance(f, str):
                 raise ValueError('flag ' + str(f) + ' is not a string')
         for f in flags:
             if f not in self.current_flags:
                 self.current_flags.append(f)
 
     def removeFlags(self, flags):
-        if type(flags) is not list:
+        if not isinstance(flags, list):
             flags = [flags]
         # Two loops intentionally. First one is for
         # input validation.
         for f in flags:
-            if type(f) is not str:
+            if not isinstance(f, str):
                 raise ValueError('flag ' + str(f) + ' is not a string')
         for f in flags:
             if f in self.current_flags:
                 self.current_flags.remove(f)
 
     # Adding and removing builds can't be done atomically.  Wondering whether
-    def addBuildsDirect(self, buildlist, release, **kwargs):
+    def addBuildsDirect(self, buildlist, default_product_version, **kwargs):
```

Comment:
I like this idea although this changes the API signature for this method.

@yazug @lhh can you please confirm that you're not using `addBuildsDirect()` in your code? (only `addBuilds()`)?

Reply:
oh, about the name I can move it back to `release` although I don't like the term `release`.
I am trying to do none breaking change about `addBuildsDirect` and `addBuilds` base on the type of `buildlist` and value of `release`.
so @yazug @lhh let me know if something isn't appropriate.

Reply:
We do use addBuildsDirect by position, not by key-value parameters

Reply:
we are using addBuildsDirect() but only by position parameters when we are looping through adding builds to the advisory

Reply:
Also we should add doctext to addBuildsDirect for how things are being passed in and make sure to add unittests for existing and anticipated calling modes.